src/relayserver/RelayServer.ts

```diff
@@ -188,17 +199,23 @@ export class RelayServer extends EventEmitter {
     throw new Error(`Refusing to serve transactions for paymaster at ${paymaster}: ${message}`)
   }
 
-  async validatePaymasterGasLimits (req: RelayTransactionRequest): Promise<{
+  async validatePaymasterGasAndDataLimits (req: RelayTransactionRequest): Promise<{
     maxPossibleGas: number
-    acceptanceBudget: number
+    relayExposure: number
   }> {
     const paymaster = req.relayRequest.relayData.paymaster
-    let gasLimits = this.trustedPaymastersGasLimits.get(paymaster)
-    let acceptanceBudget: number
-    if (gasLimits == null) {
+    let gasAndDataLimits = this.trustedPaymastersGasAndDataLimits.get(paymaster)
+    let relayExposure: number
+
+    const maxBlockGas = 12e6
```

Comment:
hey hey hey, this is a magic number! please use server parameters to define something like that.
as an option, can ask for block limit on a latest block when initializing

Reply:
Right, though it's just a place holder for uint256 as parameter, since we only care about `msg.data.length` here.